File: app/src/main/java/nz/co/codebros/quakesnz/QuakesNZApplication.java
package nz.co.codebros.quakesnz;

import dagger.android.AndroidInjector;
import dagger.android.support.DaggerApplication;

/**
 * Created by Leandro on 25/07/2014.
 */
public class QuakesNZApplication extends DaggerApplication {

    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerApplicationComponent.builder().create(this);
    }
}

File: app/src/main/java/nz/co/codebros/quakesnz/QuakesNZApplication.kt
package nz.co.codebros.quakesnz

import dagger.android.AndroidInjector
import dagger.android.support.DaggerApplication

class QuakesNZApplication : DaggerApplication() {
    override fun applicationInjector(): AndroidInjector<out DaggerApplication> =
            DaggerQuakesNZComponent.builder().create(this)
}
